tests/integration/cmd_push_test.go

```diff
@@ -53,6 +53,32 @@ var _ = Describe("odo push command tests", func() {
 			helper.HttpWaitFor("http://"+url, "UPDATED!", 30, 1)
 		})
 
+		It("should be able to create a file, push, delete, then push again propagating the deletions", func() {
+			helper.CmdShouldPass("git", "clone", "https://github.com/openshift/nodejs-ex", context+"/nodejs-ex")
```

Comment:
Why not add this in the examples folder and copy it from there?

Reply:
@mik-dass It's how we do it in the other examples :)

Reply:
@mik-dass It's ok. The spec is validated against git component.